### ISSUE TITLE
'softmax' activation function

### ISSUE THREAD

**Issue by egmaminta** [jrzaurin/pytorch-widedeep#65]
Good day! I was wondering if we could use 'softmax' activation function for classification problems? I was trying to use TabResnet and FTTransformer for multi-class classification problem and the last layer is a linear. I was hoping if I can apply 'softmax'? or is it automatically implied?

Thanks.

**Comment by jrzaurin** [comment]
hey hey @egmaminta 

The `Trainer` will take care of it, it will take the output of that last layer and apply `Softmax` if needed. Here: 

1. for the metrics: https://github.com/jrzaurin/pytorch-widedeep/blob/e1e6a2acd7634b3e9667f14741d6fac76dea7079/pytorch_widedeep/training/trainer.py#L1175
2. the loss: https://github.com/jrzaurin/pytorch-widedeep/blob/e1e6a2acd7634b3e9667f14741d6fac76dea7079/pytorch_widedeep/training/_trainer_utils.py#L207 , where the `nn. CrossEntropyLoss ` is used, equivalent to the combination of `LogSoftmax` and `NLLLoss`
